master/boltons/meta.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: boltons
+  version: {{ environ['GIT_DESCRIBE_TAG'] }}.post{{ environ['GIT_DESCRIBE_NUMBER'] }}
+
+source:
+  git_url: https://github.com/mahmoud/boltons.git
+  git_rev: master
+
+build:
+  number: 1
+
+  # Note that this will override the default build string with the Python
+  # and NumPy versions
+  string: {{ environ.get('GIT_BUILD_STR', '') }}_np{{ np }}py{{ py }}

```

Comment:
remove the `np` bit from this as boltons does not depend an numpy so we do not want to pin this to a specific numpy version.